sdk/assemblyscript/src/assembly/auth.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2024 Hypermode Inc.
+ * Licensed under the terms of the Apache License, Version 2.0
+ * See the LICENSE file that accompanied this code for further details.
+ *
+ * SPDX-FileCopyrightText: 2024 Hypermode Inc. <hello@hypermode.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { JSON } from "json-as";
+
+export function getJWTClaims<T>(): T {
+  const claims = process.env.get("JWT_CLAIMS");
+  if (!claims) {
+    throw new Error("JWT claims not found.");
+  }
```

Comment:
Since `catch` isn't supported in AssemblyScript, throwing here leaves no route for a user to test if claims are present.  How about this instead:

```suggestion
  if (!claims) {
    console.warn("JWT claims not found");
    return instantiate<T>();
  }
```

That returns their claims object with fields at empty/default values.

The other option would be to make the function return `T | null` and return null, but then the user would always have to check for null when making the call.